docs/packages/package_main.rst

```diff
@@ -3,46 +3,26 @@
 Mesa Packages 
 ==============
 
-*Supporting MESA Agent Based Modeling with Crowd Sourced Packages*  
-------------------------------------------------------------------
+		*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
+		This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
+		complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
+		(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
 
-	*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
-	This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
-	complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
-	(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
+		Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
+		(Nov. 11, 2005), pp. 987-991
 
-	Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
-	(Nov. 11, 2005), pp. 987-991
 
 
-Mesa Packages Purpose: 
-----------------------
+The Mesa core functionality is just a subset of what we believe researchers creating Agent Based Models (ABMs) will use. We designed Mesa to be extensible, so that individuals from various domains can build and maintain their own Mesa packages in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."
 
+The purpose of this guide is to support new programmers by providing detailed guides on "How to" build and share a package. Let's get started!
 
-**MESA Packages is a repository to share packages for use in Agent Based Models (ABMs) in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."**
+	You want create and share your package but have never done it before -- start here: :ref:`package-development`
```

Comment:
Lines 20-22 are a lot of words that don't need to be there. 
How about... 

Choose your adventure: 
* Learn how to create and share your package
* Learn how to incorporate a package from a Mesa user into your model